source/blog/2018-05-04-the-emberjs-times-issue-45.md

```diff
@@ -20,9 +20,11 @@ Here's a recap of what happened in Emberland these days ✨
 
 ---
 
-## [BROCCOLI JS TUTORIAL TITLE HERE](your-main-url-here)
+## [Listen to your Broccoli and it will tell you how to eat it](your-main-url-here)
+
+You might already know that Broccoli takes care of your asset pipeline. But do you know **how** it works? [Oli Griffiths](https://twitter.com/oligriffiths) has made an [amazing tutorial](http://www.oligriffiths.com/broccolijs/) on *the asset pipeline for ambitious applications*. It takes you through the basic setup and guide you through the journey of sass preprocessing, debugging and fingerprints.
```

Comment:
I think it would be 

[...] the basic setup and guide**s** you through the [...]

correct?